### PR TITLE
docs: auto generate pouch cli/api docs via code

### PR DESCRIPTION
**Pull request by pouchrobot**
Signed-off-by: pouchrobot <pouch-dev@alibaba-inc.com>

**1.Describe what this PR did**
This PR is automatically done by AI-based collaborating robot.
Pouchrobot will auto-generate cli/api document via https://github.com/spf13/cobra/tree/master/doc every day.
	
**2.Does this pull request fix one issue?** 
None

**3.Describe how you did it**
First, execute command "make client" to build new pouch cli;
Second, execute command "./pouch gen-doc" to generate new cli docs. 

For API part, we use a tool swagger2marup to make it.

**4.Describe how to verify it**
None

**5.Special notes for reviews**
The cli/api doc must be automatically generated.

### CODE REVIEW

File: docs/commandline/pouch_rename.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## pouch rename
+
+Rename a container with newName
+
+### Synopsis
+
+Rename a container object in Pouchd. You can change the name of one container identified by it's name or ID. The container you renamed is ready to be used by it's new name.
```
Comment (Letty5411) [inline docs/commandline/pouch_rename.md]: it's ->its

**Comment by Letty5411** [comment]
This PR LGTM. The typo could be fixed in source code.

File: docs/commandline/pouch_rename.md (hunk 1 above)
Comment (allencloud) [reply docs/commandline/pouch_rename.md]: That is great. I think we need to merge this and update the source code. When reviewing this pr ran into something wrong, this PR could be used to track issue, then we can arrange resource to fix these issue. 

This part is introduced by you. Could you please submit another PR to fix this? @zeppp

**Comment by allencloud** [comment]
LGTM as well. Could you fix what @Letty5411 mentioned in commit comments? @zeppp

**Pull request merged**